pkg/injector/service/config.go

```diff
@@ -204,3 +227,15 @@ func isTruthyDefaultTrue(val string) bool {
 	}
 	return kitutils.IsTruthy(val)
 }
+
+func parseStringToInt64Pointer(intStr string) (*int64, error) {
```

Comment:
Should this be in **dapr/kit**?

Reply:
Thanks for the suggestion. The conversion here is very specific to this use case. I've searched through the `dapr/dapr` code for a similar conversion of `string` to `int64` pointer, and all I've found is regular `string` to `int` (Atoi) conversion. 
At this point, I don't see a good reason to add this to `dapr/kit`, but we might in the future if more use cases will appear.